inc/classes/subscriber/Tools/class-cache-dir-size-check-subscriber.php

```diff
@@ -89,8 +89,7 @@ public function cache_dir_size_check() {
 		$current_blog_id = get_current_blog_id();
 
 		$checks = [
```

Comment:
We're touching this file. It should move into our new architecture.

Reply:
I think it would fit within the new `HealthCheck` class I added in #2544

Reply:
Then I'll wait for #2544 to be merged.

Reply:
I did the update for this, if you can both check everything one more time

Reply:
Thanks Remy, it seems fine and the tests pass.